common.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const submodules = [
+  'auth', // Login/password validation
```

Comment:
I think it is possible to use the functionality implemented here for validation of something other than logins and passwords that is also authentication/authorization related.